ts/main.ts

```diff
@@ -30,6 +30,90 @@ $form?.addEventListener('submit', (event: Event) => {
   data.nextEntryId++;
   data.entries.unshift(obj);
 
+  const newList = renderEntry(obj);
+
+  const theFirstChild = $ul?.firstChild as HTMLElement;
+  $ul?.insertBefore(newList, theFirstChild);
```

Comment:
Not needed! You can use `prepend` to add it to the beginning without needing to find the `firstChild`